src/lib/packet.h

```diff
@@ -355,6 +355,7 @@ typedef enum {
     PGP_PKA_RESERVED_DH = 21,             /* Reserved for Diffie-Hellman
                                            * (X9.42, as defined for
                                            * IETF-S/MIME) */
+    PGP_PKA_EDDSA = 22,                   /* EdDSA */
```

Comment:
Can we add link at comment to https://tools.ietf.org/html/draft-koch-eddsa-for-openpgp-04#section-8 because I don't see EDSA at RFC4880 9.1 and found here.

Logical I accept it but the constant may change when `EdDSA for OpenPGP` will be approved.

Or we can move to one of `Private/Experimental algorithm` constant here.